rre-search-platform/rre-search-platform-elastic-search-impl/src/main/java/io/sease/rre/search/api/impl/Elasticsearch.java

```diff
@@ -97,6 +108,7 @@ public void beforeStart(final Map<String, Object> configuration) {
                 .put("path.logs", logsFolder.getAbsolutePath())
                 .put("path.data", dataFolder.getAbsolutePath())
                 .put("cluster.name", "rre_" + System.currentTimeMillis());
+
```

Comment:
I'd be inclined to move the RRENode instantiation/settings build into its own method, so you don't need to do the whole beforeStart() method when you call from ExternalElasticsearch - eg. `buildRreNode(configuration)` or simiilar. That will avoid redundant folders being created.